tests/integration_tests/performance/test_cpu_template_benchmark.py

```diff
@@ -24,7 +24,7 @@
 
 BASELINES = {
     "Intel": {
-        "deserialize": {"target": 0.025, "delta": 0.02},  # milliseconds
+        "deserialize": {"target": 0.005, "delta": 0.02},  # milliseconds
```

Comment:
I understand the change in test coverage but I don't think we need this change, the test failure related to this could have been an intermittent failure. Could you please revert this particular change in test_cpu_template_benchmark.py to confirm the suspicion?

Reply:
This is indeed an intermittent failure - I am looking into it as we speak, but just trying the test once or twice without this change should work

Reply:
Done